Communityhub/ConsoleExtension/Open in Windows Admin Center.json

```diff
@@ -0,0 +1,7 @@
+{
+  "downloadLocation": "https://github.com/kwygant/WAC-MECM-RCT/blob/main/compiled%20cab/WACRCT.cab",
+  "FileHash": "1F251819C122ED4439764B27EDBF8E202A7989034BE1D7F5487D54B886CED6C9",

```

Comment:
The hash check is failing on your pull request. You'll want to re-verify the hash and update.